dydx/dydxPresenters/dydxPresenters/_v4/MarketInfo/Components/dydxMarketPositionViewPresenter.swift

```diff
@@ -84,5 +77,70 @@ class dydxMarketPositionViewPresenter: HostedViewPresenter<dydxMarketPositionVie
         viewModel?.closePrice = dydxFormatter.shared.dollar(number: position.exitPrice?.doubleValue, digits: configs.displayTickSizeDecimals?.intValue ?? 0)
 
         viewModel?.funding = SignedAmountViewModel(amount: position.netFunding?.doubleValue, displayType: .dollar, coloringOption: .allText)
+
+        // hide for now until feature work complete
+        #if DEBUG
+        let routeToTakeProfitStopLossAction = {[weak self] in
+            if let marketId = self?.position?.id {
+                Router.shared?.navigate(to: RoutingRequest(path: "/trade/take_profit_stop_loss", params: ["marketId": "\(marketId)"]), animated: true, completion: nil)
+            }
+        }
+        let routeToOrdersAction = {
+            Router.shared?.navigate(to: RoutingRequest(path: "/market", params: ["currentSection": "orders"]), animated: true, completion: nil)
+            return
+        }
+
+        let takeProfitOrders = subaccountOrders.filter { (order: SubaccountOrder) in
```

Comment:
On Android, I moved those logic to a [state extension](https://github.com/dydxprotocol/v4-native-android/blob/7c49afac065fd720445b6ffe133e42ecbac77ac5/v4/integration/dydxStateManager/src/main/java/exchange/dydx/dydxstatemanager/AbacusStateTriggerOrder.kt#L14), which I think helps with sharing the logic.

Reply:
mmm smart. I will do that in next PR